apps/ios/Shared/Views/Chat/ChatInfoView.swift

```diff
@@ -191,97 +191,109 @@ struct ChatInfoView: View {
                         // } else if developerTools {
                         //     synchronizeConnectionButtonForce()
                         // }
+
+                        NavigationLink {
+                            ChatWallpaperEditorSheet(chat: chat)
+                        } label: {
+                            Label("Chat theme", systemImage: "photo")
+                        }
+                        //                    } else if developerTools {
+                        //                        synchronizeConnectionButtonForce()
+                        //                    }
                     }
                     .disabled(!contact.ready || !contact.active)
```

Comment:
why is this removed? updating preferences would not send message at this point and preferences would diverge. same for ratchet if button is shown. verifying code also doesn't make much sense at this point if contact is not even online. receipts is the only one that could be useful but also limited use case

Reply:
it duplicates "disabled" on parent